apollo-runtime/src/jvmMain/kotlin/com/apollographql/apollo3/network/OkHttpExtensions.kt

```diff
@@ -0,0 +1,48 @@
+package com.apollographql.apollo3.network
+
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.network.http.HttpNetworkTransport
+import com.apollographql.apollo3.network.http.OkHttpEngine
+import com.apollographql.apollo3.network.ws.OkHttpWebSocketEngine
+import com.apollographql.apollo3.network.ws.WebSocketNetworkTransport
+import okhttp3.Call
+import okhttp3.OkHttpClient
+
+/**
+ * Configures the [ApolloClient] to use the [OkHttpEngine] for network requests.
+ * The [OkHttpClient] will be used for both HTTP and WebSocket requests.
+ *
+ * See also [ApolloClient.Builder.httpEngine] and [ApolloClient.Builder.webSocketEngine]
+ */
+fun ApolloClient.Builder.okHttpClient(okHttpClient: OkHttpClient) {
+  httpEngine(OkHttpEngine(okHttpClient))
+  webSocketEngine(OkHttpWebSocketEngine(okHttpClient))
```

Comment:
It would make sense to return `this` (same remark for the following extensions). (Probably use `apply`)

Reply:
Good catch 👍